blockchain/blockchain.go

```diff
@@ -881,36 +883,71 @@ func removeTxsAndReceipts(txn db.Transaction, blockNumber, numTxs uint64) error
 	return nil
 }
 
+func storeEmptyPending(txn db.Transaction, latestHeader *core.Header) error {
+	receipts := make([]*core.TransactionReceipt, 0)
+	pendingBlock := &core.Block{
+		Header: &core.Header{
+			ParentHash:       latestHeader.Hash,
+			SequencerAddress: latestHeader.SequencerAddress,
+			Timestamp:        latestHeader.Timestamp + 1,
```

Comment:
I am having second thoughts about this approach.

Normally, pending block timestamp never changes. But people may observe a change in the timestamp when we replace the empty placeholder with the actual state of the pending block from the sequencer.

The value of timestamp might affect execution of a transaction as well.

Same goes for GasPrice as well. If we return a fee estimate for a pending block request by using latest.GasPrice and the GasPrice increases drastically, the user txn might fail.

Do you think these are relevant concerns or am I overthinking it?